docs/src/appendix/scarb-toml.md

```diff
@@ -0,0 +1,236 @@
+```toml
+[tool.snforge]
+exit_first = true
+```
+
+#### `fuzzer_runs`
+The `fuzzer_runs` field specifies the number of runs of the random fuzzer.
+
+#### `fuzzer_seed`
+The `fuzzer_seed` field specifies the seed for the random fuzzer.
+
+#### Example of fuzzer configuration
+```toml
+[tool.snforge]
+fuzzer_runs = 1234
+fuzzer_seed = 1111
+```
+
+### `[[tool.snforge.fork]]`
+```toml
+[[tool.snforge.fork]]
+# ...
+```
+Allows to configure forked tests. While in use, all below fields must be set at the same time.
+
+#### `name`
+The `name` field specifies the name of the fork.
+```toml
+[[tool.snforge.fork]]
+name = "SOME_NAME"
+```
+
+#### `url`
+The `url` field specifies the address of RPC provider.
+```toml
+[[tool.snforge.fork]]
+url = "http://your.rpc.url"
+```
+
+#### `block_id.<tag|number|hash>`
+The `block_id` field specifies the block to fork from. It can be specified by `tag`, `number` or `hash`.
+```toml
+[[tool.snforge.fork]]
+block_id.hash = "0x123"
+```
+
+#### Example configuration with two forks
+```toml
+[[tool.snforge.fork]]
+name = "SOME_NAME"
+url = "http://your.rpc.url"
+block_id.tag = "latest"
+
+[[tool.snforge.fork]]
+name = "SOME_SECOND_NAME"
+url = "http://your.second.rpc.url"
+block_id.number = "123"
+```
+
+### `[profile.<dev|release>.cairo]`
+Adjust Cairo compiler configuration parameters when compiling this package. These options are not taken into consideration when this package is used as a dependency for another package. All fields are optional.
+
+```toml
+[profile.dev.cairo]
+# ...
+```
+
+#### `unstable-add-statements-code-locations-debug-info`
+If enabled, during the project compilation Scarb will add a mapping between Sierra statement indexes and locations in the code to debug info. A statement index maps to a vector consisting of code fragment which caused the statement to be generated and all code fragments that were inlined or generated along the way. By default, this flag is disabled.
+
+```toml
+[profile.dev.cairo]
+unstable-add-statements-code-locations-debug-info = true
+```
+
+#### `unstable-add-statements-functions-debug-info`
+If enabled, during the project compilation Scarb will a add mapping between Sierra statement indexes and vectors of fully qualified paths of Cairo functions to debug info. A statement index maps to a vector consisting of a function which caused the statement to be generated and all functions that were inlined or generated along the way. By default, this flag is disabled.
+```toml
+[profile.dev.cairo]
+unstable-add-statements-functions-debug-info = true
+```
+
+#### `inlining-strategy`
+This field is responsible for setting the inlining strategy to be used by compiler when building the package. The possible values are `default` or `avoid`. If avoid strategy is set, the compiler will only inline function annotated with `#[inline(always)]` attribute.
+```toml
+[profile.dev.cairo]
+inlining-strategy = "avoid"
+```
```

Comment:
Maybe let's just link to specific sections of scarb docs here instead of maintaining these ourselves?